jetty-websocket/websocket-javax-server/src/main/java/org/eclipse/jetty/websocket/javax/server/internal/JavaxWebSocketCreator.java

```diff
@@ -159,8 +159,6 @@ public Object createWebSocket(ServerUpgradeRequest req, ServerUpgradeResponse re
             // [JSR] Step 6: create endpoint class
             Class<?> endpointClass = config.getEndpointClass();
             Object endpoint = config.getConfigurator().getEndpointInstance(endpointClass);
-            // Do not decorate here (let the Connection and Session start first)
-            // This will allow CDI to see Session for injection into Endpoint classes.
```

Comment:
Why was this comment removed?
Seems like an important reminder for future devs.

Reply:
The comment is incorrect. We don't support the injection of the WebSocket Session into Endpoint classes.

We don't decorate there because we decorate in the `FrameHandlerFactory` instead, and we do this because it can be done in the same way between the Jetty and Javax sides because you can provide your own creator on the Jetty side so we cannot decorate there.